content/src/controller/Controller.ts

```diff
@@ -142,8 +143,7 @@ export class Controller {
     // Path: /entities/currently-pointed/{urnPrefix}
     const urnPrefix: string = req.params.urnPrefix
 
-    const regex = /^[a-zA-Z0-9_.:,-]+$/g
-    if (!regex.test(urnPrefix)) {
+    if (!(await isUrnPrefixValid(urnPrefix))) {
       return res
         .status(400)
         .send({ errors: `Invalid Urn prefix param: '${urnPrefix}'` })
```

Comment:
Could we here a more detailed error message like `Urn prefix param should be a valid urn prefix of a 3rd party collection` or something like that? (Assuming the name of the endpoint can't be changed)